aspnet/mvc/overview/getting-started/recommended-resources-for-mvc.md

```diff
@@ -40,7 +40,7 @@ by [Rick Anderson](https://github.com/Rick-Anderson)
 - [Selecting Cultures for Localization in ASP.NET](http://weblog.west-wind.com/posts/2014/Mar/27/Auto-Selecting-Cultures-for-Localization-in-ASPNET)
 - [Nesting Layouts in MVC](http://astutelogic.com/nesting-layouts-in-mvc/) by Russell Waite
 - [Create an ASP.NET MVC app with auth and SQL DB and deploy to Azure App Service](https://www.windowsazure.com/en-us/develop/net/tutorials/web-site-with-sql-database/)
```

Comment:
@scottaddie This tutorial points to ASP.NET Core page and maybe it should be removed (seems to be redundant)

Reply:
Please remove this one.

Reply:
Removed :)